Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM tiangolo/uvicorn-gunicorn:python3.9
```

Comment:
I suggest you always stick to [official python containers](https://hub.docker.com/_/python). You never know what other people's containers may have. Use the -slim versions.

Reply:
Even when this one is the official docker image from the tiangolo - fastapi team?
Here is the base image tiangolo/uvicorn-gunicorn:python3.9:
https://github.com/tiangolo/uvicorn-gunicorn-docker/blob/master/docker-images/python3.9.dockerfile